Cargo.toml

```diff
@@ -331,11 +330,11 @@ jsonrpsee-core = "0.20"
 jsonrpsee-types = "0.20"
 
 # crypto
-secp256k1 = { version = "0.27.0", default-features = false, features = [
+secp256k1 = { version = "0.28", default-features = false, features = [
     "global-context",
     "recovery",
 ] }
-enr = { version = "=0.10.0", default-features = false, features = ["k256"] }
+enr = { version = "0.11", default-features = false, features = ["k256"] }
```

Comment:
```suggestion
enr = { version = "0.11", default-features = false, features = ["rust-secp256k1"] }
```

Reply:
looks like both are needed since without k256 it fails to compile: https://github.com/sigp/enr/pull/74